lib/barong/app.rb

```diff
@@ -57,6 +57,15 @@ def type!(key, value, options)
         when :integer
           regex!(key, value,  /^\d+$/)
           return value
+
+        when :path
+          return Rails.root.join(value).tap { |p| path!(key, p) }
+        end
+      end
+
+      def path!(key, path)
```

Comment:
we also have path in barong, like path to seeds, path to label_state config.
Update them to :path type please